Passengerfile.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
The logs were going to the wrong places (I've already established how to get logs to rotate on the server, and the default passenger location wouldn't do that easily), and I wanted to ensure that there were always at least 2 instances running...